src/ddsrt/include/dds/ddsrt/ifaddrs.h

```diff
@@ -13,16 +13,23 @@
 
 #include "dds/ddsrt/sockets.h"
 
+/**
+  @file ifaddrs.h
+  @brief This header deals with interface addresses
+*/
+
 #if defined (__cplusplus)
 extern "C" {
 #endif
 
+/** @brief The interface type */
 enum ddsrt_iftype {
   DDSRT_IFTYPE_UNKNOWN,
```

Comment:
These are pretty self-explanatory, but I think the documentation "standard" includes documenting the values. Same holds for documenting the fields of `struct ddsrt_ifaddrs`.

Reply:
Very well then, it's just that I had a gut feeling that these would be over the top in the sense of pointing out the obvious (I know you have a different standard for what's obvious, but that would result in even less comments).